ui/components/src/Table/Table.stories.tsx

```diff
@@ -116,9 +119,11 @@ export const Basic: Story = {
  * - `header`: string to display in the header to identify that column.
  *
  * Each column config may include the following optional properties:
- * - `width`: determines the size of the column in the table. It can be set to a number indicating a pixel width or to "auto."
  * - `align`: determinse how the content in the column's cells are aligned. Can be set to: `left`, `right`, or `center`.
  * - `cell`: function used to customize rendering of the content in a given column cell. Note that cells have a hardcoded height your customization must work within.
+ * - `cellDescription`: Text to display when hovering over a cell. This can be useful for providing additional information about the column when the content is ellipsized to fit in the space. If set to `true`, it will use the value generated by the `cell` prop if it can be treated as a string.
```

Comment:
The `true` case feels slightly awkward, but it's really nice for the use case where you want to display the same thing as the cell content without doing the calculations twice (once in `cell` and once in `cellDescription`).